felix/labelindex/label_inheritance_index.go

```diff
@@ -148,10 +150,13 @@ func (l *InheritIndex) OnUpdate(update api.Update) (_ bool) {
 			log.Debugf("Deleting host endpoint %v from InheritIndex", key)
 			l.DeleteLabels(key)
 		}
-	case model.ProfileLabelsKey:
+	case model.ResourceKey:
```

Comment:
According to this change, older versions of Felix will still be expecting the ProfileLabelsKey to be sync'd from Typha.

Reply:
Addressed by #5695